apps/i18n/common/en_us.json

```diff
@@ -297,6 +298,7 @@
   "submitYourProject": "Submit your project",
   "submitYourProjectConfirm": "You cannot edit your project after submitting it, really submit?",
   "teacherPanel": "Teacher Panel",
+  "tryNow": "Try Now",
```

Comment:
According to Tanya, because this string exists in dashboard, we'll get translations for it for free once the translation sync script runs :)